### ISSUE TITLE
请问其中的虚拟人形象可以自己更改吗？人脸中的关键帧还需要自己设置吗？

### ISSUE THREAD

**Issue by orangeKillery** [FACEGOOD/FACEGOOD-Audio2Face#33]

**Comment by SZNero** [comment]
Yes,you can change the character in UE. But ,if you do that things ,you need to make sure the character has the same blendshapes as the model we used or you can try to transform the weights to suit the ARKIT.

**Comment by Anne-2017** [comment]
how to  change the character in UE
